components/colorarea/stories/template.js

```diff
@@ -0,0 +1,36 @@
+// template is where the component markup with go
+
+import { html } from 'lit-html';
+import { classMap } from 'lit-html/directives/class-map.js';
+import { ifDefined } from 'lit-html/directives/if-defined.js';
+import { Template as ColorHandle } from "@spectrum-css/colorhandle/stories/template.js";
+
+import '../index.css';
+
+export const Template = ({
+  rootClass = "spectrum-ColorArea",
+  customClasses = [],
+  isDisabled = false,
+  customSize,
+  handlePosition,
+  style = "--spectrum-picked-color: rgb(255, 0, 0);" + handlePosition,
```

Comment:
Since this is being passed down to handle, maybe we can rename this property to indicate that so it doesn't get confused with the rootClass's custom styles?  Or maybe we can even pass those down as values to ColorHandle's function to render?